lua/cfc_http_restrictions/default_config.lua

```diff
@@ -1,9 +1,19 @@
 AddCSLuaFile()
 
-return {
+---@alias WhitelistAddressOption { allowed: boolean|nil, noisy: boolean|nil, permanent: boolean|nil }
+
+---@class WhitelistConfig 
+---@field version string
+---@field wrapHTMLPanels boolean|nil
+---@field defaultAssetURIOptions WhitelistAddressOption
+---@field defaultOptions WhitelistAddressOption
+---@field addresses table<string, WhitelistAddressOption>
+
+---@type WhitelistConfig
```

Comment:
I think you can just do this
```suggestion
---@class WhitelistConfig 
---@field version string
---@field wrapHTMLPanels boolean|nil
---@field defaultAssetURIOptions WhitelistAddressOption
---@field defaultOptions WhitelistAddressOption
---@field addresses table<string, WhitelistAddressOption>
```